### PR TITLE
Create GitHub Action Code

### PR DESCRIPTION
**Pull request by umisora**
GitHub Repository Graphic

## What about this ?
Coded for GitHub Action about Admina App Syncs.

## What can be done ?
We can register AzureAD SSO Apps with Admina on a CLI basis.

refs:
https://github.com/actions/typescript-action

### CODE REVIEW

**Comment by umisora** [comment]
```
> node dist/index.js

Starting sync...
Start Sync Data from AzureAd to Admina .
Getting Azure AD data...
Getting access token...
Detected 39 users in テストログインアプリ
Detected 55 users in Google Workspace
Registering custom service...
Sync finished.
Workspace already exists | ServiceName: Single Sign-On(xxxxx) ,WorkspaceName: テストログインアプリ(xxxxx)
Workspace already exists | ServiceName: Single Sign-On(xxxxx) ,WorkspaceName: Google Workspace(xxxxx)
Register data into テストログインアプリ: existingUsers 39 , newUsers 0 , deleteAccounts 0
Register data into Google Workspace: existingUsers 55 , newUsers 0 , deleteAccounts 0
```

File: .gitignore
Comment (peter-asztalos-mf) [inline .gitignore:1]: I believe we can add the lib folder here, as it only contains the compiled code which is used by the package command to generate the dist folder.

File: src/index.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      throw new Error(`Unsupported subcommand: ${subcommand}`)
+    }
+  } catch (error: any) {
+    if (error instanceof Error) {
+      core.setFailed(error.message)
+    } else {
+      core.setFailed(error)
+    }
+  }
+  console.log('Sync finished.')
```
Comment (peter-asztalos-mf) [inline src/index.ts:39]: this should be inside the sync related block
```suggestion
  console.log('Sync finished.')
```

**Comment by peter-asztalos-mf** [comment]
The whole thing is AI generated?

**Comment by umisora** [comment]
@peter-asztalos-mf 

The main code is mostly GPT. I gave a little more specific instructions than before (separate class structures, etc.) 

Ecosystem code such as npm copied the sample repository!

**Comment by umisora** [comment]
the sample repo gave us making typescript work with GitHub actions.

**Pull request merged**